pkg/feature/storage/v2/flag_trigger.go

```diff
@@ -169,7 +169,6 @@ func (f flagTriggerStorage) GetFlagTrigger(
 		&trigger.Description,
 		&trigger.TriggerCount,
 		&trigger.LastTriggeredAt,
-		&trigger.Token,
```

Comment:
We can't remove the token from the query because the Update API calls the GET query. Otherwise, it will delete the current token data when the update query is committed.

https://github.com/bucketeer-io/bucketeer/blob/aa5e4e5ddc4acbaa333ed80d26ff2833f73895e4/pkg/feature/api/flag_trigger.go#L177-L181

It will be safer to remove it at the API layer to avoid bugs.

Reply:
We need to delete the token from the List API response, too.
Also, as a default rule, remove the token from any API that returns the flag trigger. Such as create API, etc.